### ISSUE TITLE
A11y: [Send a message] Circular 'Seen by' button is not announced by scre...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-android#6393]
- **Checkpoint ID:** 11.1.3.2/8.2
- **Component:** Send a message

### Issue Description
Circular 'Seen by' button is not announced by screen reader.

### Recommendation
Expose 'Seen by' button to TalkBack with correct label/role.

--- 
*Report Date: 25.02.2026*
